composer.json

```diff
@@ -4,10 +4,20 @@
   "description": "Benchmark module",
   "license": "proprietary",
   "require": {
-    "php": ">=7.2"
+    "php": ">=7.2",
+    "phpbench/phpbench": "^0.17.0",
+    "spryker/guzzle": "^2.1.0",
+    "spryker/kernel": "^3.30.0",
+    "spryker/symfony": "^3.1.0",
+    "spryker/util-encoding": "^1.0.0 || ^2.0.0"
```

Comment:
Why we need to support both versions?

Reply:
In order not to force customers to have `util-encoding` if version 2 if they don't need it.

Reply:
See my comment below: It seems impossible to use v1 here.